app/messaging/create-submission-msg.js

```diff
@@ -219,7 +220,17 @@ function getEmailDetails (submission, desirabilityScore, notifyTemplate, agentAp
       productivityScore: getQuestionScoreBand(desirabilityScore.desirability.questions, 'Q19'),
       collaboration: submission.collaboration,
       collaborationScore: getQuestionScoreBand(desirabilityScore.desirability.questions, 'Q20'),
-      sssi: submission.sSSI
+      sssi: submission.sSSI,
+      businessName: submission.businessDetails.businessName,
+      farmerName: submission.farmerDetails.firstName,
+      farmerSurname: submission.farmerDetails.lastName,
+      agentName: submission.agentDetails.firstName ?? 'N/A',
```

Comment:
`submission.agentDetails` might not exist, so this will cause an error if you do a farmer only application. Put `?.` on the access to firstName